src/main/ts/ast/ts/DtsCleaner.ts

```diff
@@ -0,0 +1,53 @@
+import * as ts from 'typescript';
+import { serialize } from './Serializer';
+
+export interface CleanerOptions {
+  keepComments?: boolean;
+  keepVariables?: string[];
+  removeNamespaces?: string[];
+}
+
+const isNamed = (node: { name: ts.PropertyName | ts.BindingName }, names: string[]) =>
```

Comment:
Is there a node type? Can we use it, or use `Pick<TheNodeType, 'name'>`?

Reply:
No, as the `name` property only exists on specific nodes and it's not on the base Node type.